examples/dev/dev_use_unit_system.py

```diff
@@ -195,3 +196,13 @@ def thread2():
 
 # Without locks thread2 executes before thread 1
 print(f"After running both threads: {threaded_data.m}")
+
+schema = fl.SI_unit_system.schema()
+
+print(schema)
+
+defaults = UnitDefaults()
+
+schema = defaults.schema()
```

Comment:
we don't need schema for it, right? Just `defaults.dict()`

Reply:
Correct, currently only a dict is exported in the script.